registry/src/config.rs

```diff
@@ -1 +1,14 @@
 // (c) Copyright 2023 Helsing GmbH. All rights reserved.
+use clap::Parser;
+use std::net::SocketAddr;
+
+#[derive(Parser, Clone, Debug)]
+pub struct Config {
+    /// Address to listen to for incoming connections.
+    #[clap(long, short, env, default_value = "0.0.0.0:8000")]
+    pub listen: SocketAddr,
+
+    /// URL of Postgres database to connect to.
+    #[clap(long, short, env, default_value = "postgres://buffrs:buffrs@localhost")]
```

Comment:
no default please 👀

Reply:
Because of the risk of forgetting to set it in production?

Reply:
More because this is a weird default, when not developing (and its shown in the cli)

Reply:
Not quite sure what you mean by a weird default. I think it is quite useful to have a default value here.

What I like to be able to do is launch it locally without needing a lot of configuration. For example, this is how I can launch it locally at the moment:

```rust
docker compose up # launch database
cargo run         # launch registry
```

Removing this default means that I need to either memorize the default, or introduce some kind of runner:

```rust
docker compose up
# this is quite annoying to have to type out every time
cargo run -- --listen postgres://buffrs:buffrs@localhost
```

There is one change we could make that might work here, which is to keep it but enable it only for development builds, by specifying it like this:

```rut
#[cfg_attr(dev, clap(default_value = "postgres://buffrs:buffrs@localhost"))]
```

The advantage of this is that it means there is no default value for production. I have had an instance before where an environment variable was spelled wrong, resulting in a configuration value being set incorrectly and that was quite annoying to debug. So in doing it this way, we preserve the ability to launch it locally easily while not having a default set in the production release.

Would this idea address your concern but still allow a good developer experience?

Reply:
Yes, thank you!